### PR TITLE
feat: Productpage UI

### PR DESCRIPTION
**Pull request by ozairkhan2810**
Product with components
- Product List
- Product Filter

Files to check: .js(7 files)

**Screenshot**
image

### CODE REVIEW

File: src/pages/homepage/Categories.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const Categories = () => {
             key={category._id}
             class="category-list flex flex-column gap-1 justify-center align-center bg-white box-shadow"
           >
-            {/* <img class="img-category" src={category.categoryImg} alt={category.categoryName} /> */}
+            <img class="img-category" src={category.categoryImg} alt={category.categoryName} />
```
Comment (Mohitkumar33) [inline src/pages/homepage/Categories.js:14]: you have to change the class to className

File: src/pages/products/ProductFilter.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                    <span>1000</span>
+                </div>
+                <input
+                    type="range"
+                    id="vol"
+                    name="vol"
+                    min="100"
+                    max="1000"
+                    step="100"
+                    value="500"
```
Comment (Mohitkumar33) [inline src/pages/products/ProductFilter.js]: you can search about defaultValue={}. You will need when taking the input on change.

File: src/pages/products/ProductList.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import ProductFilter from "./ProductFilter";
+import { products } from '../../backend/db/products'
+import './productList.css'
+
+const ProductList = () => {
+    return (
+        <div class="content-container mt-1 flex gap-1">
```
Comment (Mohitkumar33) [inline src/pages/products/ProductList.js]: change class to className

**Review by shaksham08 — APPROVED** [review]
Overall Code looks clean and good

**Review by Mohitkumar33 — APPROVED** [review]

**Review by fahd23 — APPROVED** [review]
Screenshot (87)_LI

One feedback, I have attached the Screenshot above, Have a look into that.

**Pull request merged**